runtime/doc/lua.txt

```diff
@@ -1017,21 +1017,18 @@ vim.call({func}, {...})                                           *vim.call()*
         See also |vim.fn|.
         Equivalent to: >
             vim.fn[func]({...})
-<
-vim.cmd({cmd})                                                     *vim.cmd()*
-        Executes multiple lines of Vimscript at once. It is an alias to
-        |nvim_exec()|, where `output` is set to false. Thus it works identical
-        to |:source|.
-        See also |ex-cmd-index|.
-        Example: >
-            vim.cmd('echo 42')
-            vim.cmd([[
-              augroup My_group
-                autocmd!
-                autocmd FileType c setlocal cindent
-              augroup END
-            ]])
-<
+
+vim.cmd({command})
+        Executes one or multiple Vim script commands, depending on the type of
+        {command}:
+
+        * if {command} is a string (possibly containing newlines), execute it
+          as if passed to `:source`.
+        * if {command} is a table, execute it directly as a single command
+          with arguments as returned by |nvim_parse_command|.
+
+        See |vim.cmd()| for details.
```

Comment:
why do we have both? we can delete this section now

Reply:
Because one is autogenerated from a docstring (which we should push for much more than we have so far, and was missing for `vim.cmd` before), while the other (which you commented on) is "user manual" style hand-written entry. 

This is erring on the side of minimal change of adding proper docstrings, while not burninating the previous doc section.

(Long-term, we should more clearly separate the reference documentation and the Neovim user manual.)

Reply:
There is no "user manual", the reference is that. This section is here because `vim.fn` and other stuff here isn't yet auto-generated.

Reply:
Related comment from Bram from earlier today, re: "user manual" vs. "reference manual": https://github.com/vim/vim/pull/10405#issuecomment-1124282340

Reply:
Yes. It's an entire book :) Not sustainable nor necessary for us really. The "intro" sections in Nvim reference provide narrative. I don't see why we need to have a mostly redundant `vim.cmd` doc here.

Reply:
Then this whole section should go and replaced by proper docstrings (which end up in the `:help` wherever they end up).

Reply:
Maybe we could leave that to a followup PR and just remove the docs for `vim.cmd` from that section for now

Reply:
Move the tag to the docstring and just put a "see |vim.cmd|" there for now.

Reply:
> Then this whole section should go and be replaced by proper docstrings (

The generator can't find `vim.fn` . This section is for things that the generator can't find.